### PR TITLE
Update qgis server layer's xml file also.

### PR DESCRIPTION
**Pull request by ismailsunni**
Implementing @timlinux 's comment in https://github.com/kartoza/geonode/issues/143#issuecomment-306773982

Fixing #143 

I only put attribute from InaSAFE (which is the only geosafe we care). I am thinking to add more attribute, but I need to check where to put the attribute in the xml file first.

### CODE REVIEW

File: geonode/qgis_server/xml_utilities.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    for name, path in properties.iteritems():
+        if name in new_values:
+            elem = root.find(path, XML_NS)
+            if elem is None:
+                # create elem
+                elem = insert_xml_element(root, path)
+            elem.text = new_values[name]
+
+    exml.write(open(xml_file, 'w'))
```
Comment (lucernae) [inline geonode/qgis_server/xml_utilities.py]: newline here

File: geonode/qgis_server/tests/test_xml_utilities.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+#########################################################################
+
+from django.test import TestCase
```
Comment (lucernae) [inline geonode/qgis_server/tests/test_xml_utilities.py]: I think, you can use unittest.TestCase.
There is no db involved in the test case and you will benefit from the faster execution.

File: geonode/qgis_server/xml_utilities.py (hunk 1 above)
Comment (ismailsunni) [inline geonode/qgis_server/xml_utilities.py]: It has a new line. If there is no new line, there should be a red mark

Comment (lucernae) [inline geonode/qgis_server/xml_utilities.py]: Ah, yes, sorry.

**Pull request merged**